.ci/README.md

```diff
@@ -116,25 +116,24 @@ based on those labels. Possible labels are listed below:
 ### Running the CI locally
 
 The CI can run on a local VM using the following instructions:
-- Setup a Fedora VM
-  - Create a Fedora VM having at least 4G RAM and a 30GB disk.
+- Setup a Fedora/CentOS/CentOS-Stream VM
+  - Create a Fedora/CentOS/CentOS-Stream VM having at least 4G RAM and a 30GB disk.
   - Create a `sudo` user that does not require `sudo` password.
-  - Run the bash commands replacing the `fedora` user with the created one and the`CI_JOB` with the desired one:
+  - Run the bash commands replacing the `CI_JOB` with the desired one:
     ```sh
-    dnf install -y git vim make wget openssl driverctl pciutils
-    dnf remove -y qemu-guest-agent
-    wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
-    sudo -H -u fedora bash -c 'mkdir -p ~/go/src/github.com/kata-containers'
-    sudo -H -u fedora bash -c 'cd ~/go/src/github.com/kata-containers && git clone https://github.com/kata-containers/tests.git'
-    sudo -H -u fedora bash -c 'echo export GOPATH=/home/fedora/go >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo export PATH=/usr/local/go/bin:\$GOPATH/bin:\$PATH >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo export USE_DOCKER=true >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo export CI_JOB="CRIO_K8S" >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo cd ~/go/src/github.com/kata-containers/tests >> ~/.bashrc'
-    sudo -H -u fedora bash -c 'echo source .ci/ci_job_flags.sh >> ~/.bashrc'
-    grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
+    sudo dnf install -y git vim make wget openssl driverctl pciutils
```

Comment:
Why `wget` and not `curl`?

Reply:
As it was copied i supposed it's used by the scripts somewhere, I'll try to remove tit and see if it's actually required

Reply:
Thanks!